crates/miden-tx/src/host/account_delta_tracker.rs

```diff
@@ -15,10 +17,11 @@ use miden_objects::{
 /// TODO: implement tracking of:
 /// - all account storage changes.
 /// - account code changes.
-#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Were these dropped because now with the concept of normalized deltas it is more ambiguous to compare two `AccountDeltaTracker`s?